cmd/porter/mixins.go

```diff
@@ -39,6 +41,29 @@ func buildMixinsListCommand(p *porter.Porter) *cobra.Command {
 		},
 	}
 
-	cmd.Flags().StringVarP(&opts.rawFormat, "output", "o", "table", "Output format, allowed values are: table, json")
+	cmd.Flags().StringVarP(&opts.rawFormat, "output", "o", "table",
+		"Output format, allowed values are: table, json")
+
+	return cmd
+}
+
+func BuildMixinInstallCommand(p *porter.Porter) *cobra.Command {
+	opts := mixin.InstallOptions{}
+	cmd := &cobra.Command{
+		Use:   "install",
+		Short: "Install a mixin",
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return opts.Validate(args)
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return p.Mixins.Install(opts)
+		},
+	}
+
+	cmd.Flags().StringVarP(&opts.Version, "version", "v", "latest",
+		"The mixin version. This can either be a version number, or a tagged release like 'latest' or 'canary'")
+	cmd.Flags().StringVar(&opts.URL, "url", "",
```

Comment:
This may have been considered/mentioned already, but do we want to supply a default url here?

I'd think it would point to whatever our current mixin store is (as of writing, `https://deislabs.blob.core.windows.net/porter/mixins`).  Then, `porter mixin install helm` would just work.

But perhaps this isn't necessary as these same mixins that we currently track/host (`helm`, `azure`, etc.) are/will be automatically fetched for a given bundle (`porter.yaml`) that requires them...

Reply:
That's my plan for a follow-up. We can't do that immediately because so far we have been publishing our mixins into a separate directory per mixin e.g. .../helm, .../exec .../azure, so there isn't a single default URL that works.

My plan is to get to a point ASAP where the default install gives people deislabs as a default "mixin subscription" so that they get all the mixins we make. So they can type `porter mixin install helm` or `porter mixin upgrade --all` and it will be able to figure it out based on what we publish for all of our mixins.  But this PR is just the first step for installing just a single mixin from a URL.

Reply:
I'm also going to add a vanity URL so that we aren't exposing the full ugly url for our blob storage container. 😀 So very quickly it will be a default URL of porter.sh/mixins/atom.xml